packages/rrweb/src/replay/index.ts

```diff
@@ -572,9 +616,11 @@ export class Replayer {
     this.wrapper.classList.add('replayer-wrapper');
     this.config.root.appendChild(this.wrapper);
 
-    this.mouse = document.createElement('div');
-    this.mouse.classList.add('replayer-mouse');
-    this.wrapper.appendChild(this.mouse);
+    for (let [, { pointerEl }] of Object.entries(this.pointers)) {
+      pointerEl = document.createElement('div');
+      pointerEl.classList.add('replayer-mouse');
+      this.wrapper.appendChild(pointerEl);
```

Comment:
this is just `createNewTouch` right?

Reply:
yup, this is the code i modeled `createNewTouch` after